Build-FFmpeg.ps1

```diff
@@ -45,7 +45,7 @@ param(
 
     [switch] $BuildNugetPackage,
 
-    [version] $NugetPackageVersion = "1.0.0",
+    [string] $NugetPackageVersion = "1.0.0",

```

Comment:
Why not use the `version` type, which is used by NuGet already?

Reply:
Because NuGet version can contain prerelease tags, e.g. "1.0.0-pre2". This string cannot be converted to Version class.